HHS/Head-Start-TTADP
# [PROD] TTAHUB: 555 Prevent Showing Expired Grants with End Date of 2025, 548 Prevent Multiple Download Clicks

## Description of change

**TTAHUB-555:** Fix issue where inactive grants with an end date of 06/29/2025 where showing in the grants list.

**TTAHUB-548:** Prevent clicking the download reports button multiple times if the first download was still processing.


## How to test

**TTAHUB-555:** View a recipient that has an INACTIVE grant with a end date after today. These grants should no longer be shown.

**TTAHUB-548:** Attempt to download Alerts or AR's by clicking the download button multiple times. The button should be disabled while the download is happening.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-555
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-548


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
